scalableminds/webknossos
# Fix resolution datasource-properties suggestion bug

- Fix preview of recommended config differed in isotropic resolutions (`1` vs `[1,1,1]`)
- Removed internal representation of single int, instead are only using `Vec3Int`
- Also changed the proxy addresses to use `127.0.0.1` instead of `localhost` to make it work on macOS

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- change `datasource-config.json` on disk to have `1` as resolution and check in the datasource settings in webKnossos whether there are suggested changes regarding the resolution

### Issues:
- fixes #6057

------
(Please delete unneeded items, merge only when none are left open)
